### DISCUSSION TITLE
When the All times menu is mobile, there is a problem that the layout is broken.

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4937] [Plugins]
Hello.
When you access Kimai on your mobile, it looks good with a responsive layout.
But, if you access the All times menu, the layout will be broken as shown below.
@kevinpapst Please check.

![Screenshot_20240627_164647](https://github.com/kimai/kimai/assets/165639563/b008b7cb-8305-4a74-81e0-ad1e7395ffff)

**Comment by seleondevel** [comment]
The capture screen may look strange, but when viewed on mobile, it looks overflowing to the right.

**Reply by seleondevel** [reply]
![스크린샷 2024-07-01 152930](https://github.com/kimai/kimai/assets/165639563/dbf82630-58ff-4172-8e3c-4829ddece7bc)

When you configure the table column of your custom display, it overflows to the right of your mobile layout.

**Reply by kevinpapst** [reply]
Then switch to the "mobile" Profile.

If you hard configure more columns than the ones that fit, it looks like that. 
Scrolling tables cause different issues.
I haven't found a good solution that fixes all issues.

If you find one, please let me know.
